draft.xml

```diff
@@ -101,500 +88,440 @@
+        respectively: <figure>
+            <artwork><![CDATA[
     192.0.2.0/25,US,US-AL,,
     192.0.2.5,US,US-AL,Alabaster,
     192.0.2.128/25,PL,PL-MZ,,02-784
     2001:db8::/32,PL,,,
     2001:db8:cafe::/48,PL,PL-MZ,,02-784
-            </artwork>
-          </figure>
-        </t>
-
-        <t>
-Experimentally, RIPE has published geolocation information for their conference
-network prefixes, which change location in accordance with each new event.
-<xref target="GEO_RIPE_NCC"/> at the time of writing contains:
-          <figure>
-            <artwork>
-    193.0.24.0/21,IE,IE-D,Dublin,
-    2001:67c:64::/48,IE,IE-D,Dublin,
-            </artwork>
-          </figure>
-        </t>
-
-        <t>
-Similarly, ICANN has published geolocation information for their portable
-conference network prefixes.  <xref target="GEO_ICANN"/> at the time of writing
-contains:
-          <figure>
-            <artwork>
-    199.91.192.0/21,US,US-CA,Los Angeles,
-    2620:f:8000::/48,US,US-CA,Los Angeles,
-            </artwork>
-          </figure>
-        </t>
-
-        <t>
-Furthermore, it is worth noting that the geolocation data of SixXS users,
-already available at whois.sixxs.net, is now also accessible in the format
-described here (see <xref target="GEO_SIXXS"/>).  This can be particularly
-useful where tunnel broker networks <xref target="RFC3053"/> are concerned as:
-          <list style="symbols">
-            <t>
-the geolocation attributes of users with neighboring prefixes can be quite
-different and therefore not easily aggregated, and
-            </t>
-            <t>
-attempting to learn this data by statistical analysis can be complicated by
-the likely low number of samples for any given user, making satisfactory
-statistical confidence difficult to achieve.
-            </t>
-          </list>
-        </t>
+            ]]></artwork>
+          </figure></t>
+
+        <t>The IETF network publishes geolocation information for the meeting
+        prefixes, and generally just comment out the last meeting information
+        and append the new meeting information. The <xref target="GEO_IETF"/>
+        at the time of this writing contains:</t>
+
+        <t><figure>
+            <artwork><![CDATA[# IETF 104, March 2019 - Prague, CZ.
+# Note that Prague changed from CZ-PR to CZ-10 2016-11-15 - https://www.iso.org/obp/ui/#iso:code:3166:CZ
+130.129.0.0/16,CZ,CZ-10,Prague,
+2001:df8::/32,CZ,CZ-10,Prague,
+31.133.128.0/18,CZ,CZ-10,Prague,
+31.130.224.0/20,CZ,CZ-10,Prague,
+2001:67c:1230::/46,CZ,CZ-10,Prague,
+2001:67c:370::/48,CZ,CZ-10,Prague,]]></artwork>
+          </figure></t>
+
+        <t>Experimentally, RIPE has published geolocation information for
+        their conference network prefixes, which change location in accordance
+        with each new event. <xref target="GEO_RIPE_NCC"/> at the time of
+        writing contains: <figure>
+            <artwork><![CDATA[
+   193.0.24.0/21,IS,IS-1,Reykjav?k,
```

Comment:
Reykjav?k looks unintended. Fix please.

Reply:
Hmmm... this is going to be tricky -- I copied and pasted from the RIPE file, but currently it cannot be rendered in draft / RFC format... Not sure how best to address this...

Reply:
Reverted in f53834c7c98974174db2ebd2c9aad3b3d3f5132a

By the time this is published a: we should be able to do UTF-8, and / or b: the Reykjavík meeting will have passed :-)